exporter/opentelemetry-exporter-prometheus-remote-write/README.rst

```diff
@@ -0,0 +1,322 @@
+OpenTelemetry Prometheus Remote Write Exporter
+=========================================================
+
+This package contains an exporter to send `OTLP`_ metrics from the
+`OpenTelemetry Python SDK`_ directly to a `Prometheus Remote Write integrated backend`_
+(such as Cortex or Thanos) without having to run an instance of the
+Prometheus server. The latest `types.proto`_ and `remote.proto`_
+protocol buffers are used to create the WriteRequest. The image below shows the
+two Prometheus exporters in the OpenTelemetry Python SDK.
+
+Pipeline 1 illustrates the setup required for a `Prometheus "pull" exporter`_.
+
+Pipeline 2 illustrates the setup required for the Prometheus Remote
+Write exporter.
+
+|Prometheus SDK pipelines|
+
+The Prometheus Remote Write Exporter is a "push" based exporter and only
+works with the OpenTelemetry `push controller`_. The controller
+periodically collects data and passes it to the exporter. This exporter
+then converts the data into `timeseries`_ and sends it to the Remote
+Write integrated backend through HTTP POST requests. The metrics
+collection datapath is shown below:
```

Comment:
This comment is quite out of date now, should be `PeriodicExportingMetricReader`.